apps/agreement/test/agreement/agreement_evidence.js

```diff
@@ -91,26 +91,28 @@ contract('Agreement', ([_, someone, submitter, challenger]) => {
                 })
 
                 context('when the dispute was not ruled', () => {
-                  const itSubmitsEvidenceProperly = from => {
+                  const itSubmitsEvidenceProperly = (from, hadFinishedSubmittingEvidence) => {
                     const itRegistersEvidenceProperly = finished => {
                       const evidence = '0x123123'
+                      const hasFinished = finished && !hadFinishedSubmittingEvidence
```

Comment:
Kind of confused about this variable... I notice that it only updates some tests' names but not sure why it's a xor

Reply:
Fixed now in ac9ada497f10e6edd9a72a17c4b9839f498c34f0